### PR TITLE
Others .... [0.61.1]

### PR DESCRIPTION
**Pull request by Furtif**
0.69.1 supports

#### [Logic lasted changes](https://github.com/Necrobot-Private/PoGo.NecroBot.Logic/pull/38)

### CODE REVIEW

**Comment by Lord-Haji** [comment]
Update submods

**Comment by Furtif** [comment]
is no ok version number is 2.13.0 real v number at aeonlucid

**Comment by hardknoxni69a** [comment]
needs to be updated to test correctly and some of the changes should be Berries

**Comment by Furtif** [comment]
@hardknoxni69a  Where updated?
this ? https://github.com/Necrobot-Private/NecroBot/pull/1924/commits/fbdef9a3d1393f769db3fa64240f16ff614f6401

File: PoGo.NecroBot.CLI/Config/Translations/TEMPLATE-translation.en.json [hunk 1]
```diff
@@ -402,7 +402,7 @@
     },
     {
       "Key": "currentMiscItemInv",
-      "Value": "Berries:   {0,3:##0} | Incense:      {1,3:##0} | LuckyEggs:    {2,3:##0} | Lures:       {3,3:##0}"
+      "Value": "Berries:   {0,3:##0} | Incense:      {1,3:##0} | LuckyEggs:    {2,3:##0} | Lures:       {3,3:##0} | Evulotion:       {4,3:##0}"
```
Comment (Lord-Haji) [inline PoGo.NecroBot.CLI/Config/Translations/TEMPLATE-translation.en.json]: Here and other translations

Comment (Furtif) [inline PoGo.NecroBot.CLI/Config/Translations/TEMPLATE-translation.en.json]: yes added evolutions items

Comment (Furtif) [inline PoGo.NecroBot.CLI/Config/Translations/TEMPLATE-translation.en.json]: one bug is present not ok is berries count :s

**Pull request merged**